python/cudf/cudf/core/series.py

```diff
@@ -2788,6 +2788,12 @@ def cov(self, other, min_periods=None):
 
         return lhs._column.cov(rhs._column)
 
+    def transpose(self):
```

Comment:
Should we also add a property `T` that acts as an alias for this function?

Reply:
I think that would be a good idea and would also match the `Dataframe.transpose` API that we have exposed. Thanks!